perception/raindrop_cluster_filter/raindrop_cluster_filter.md

```diff
@@ -0,0 +1,54 @@
+
+| Name           | Type                                                     | Description            |
+| -------------- | -------------------------------------------------------- | ---------------------- |
+| `input/object` | `tier4_perception_msgs::msg::DetectedObjectsWithFeature` | input detected objects |
+
+### Output
+
+| Name            | Type                                                     | Description               |
+| --------------- | -------------------------------------------------------- | ------------------------- |
+| `output/object` | `tier4_perception_msgs::msg::DetectedObjectsWithFeature` | filtered detected objects |
+
+## Parameters
+
+### Core Parameters
+
+| Name                              | Type  | Default Value                                           | Description                                   |
+| --------------------------------- | ----- | ------------------------------------------------------- | --------------------------------------------- |
+| `filter_target_label.UNKNOWN`     | bool  | false                                                   | If true, unknown objects are filtered.        |
+| `filter_target_label.CAR`         | bool  | false                                                   | If true, car objects are filtered.            |
+| `filter_target_label.TRUCK`       | bool  | false                                                   | If true, truck objects are filtered.          |
+| `filter_target_label.BUS`         | bool  | false                                                   | If true, bus objects are filtered.            |
+| `filter_target_label.TRAILER`     | bool  | false                                                   | If true, trailer objects are filtered.        |
+| `filter_target_label.MOTORCYCLE`  | bool  | false                                                   | If true, motorcycle objects are filtered.     |
+| `filter_target_label.BICYCLE`     | bool  | false                                                   | If true, bicycle objects are filtered.        |
+| `filter_target_label.PEDESTRIAN`  | bool  | false                                                   | If true, pedestrian objects are filtered.     |
+| `max_x`                           | float | 60.00                                                   | Maximum of x of the filter effective range    |
+| `min_x`                           | float | -20.00                                                  | Minimum of x of the filter effective range    |
+| `max_y`                           | float | 20.00                                                   | Maximum of y of the filter effective range    |
+| `min_y`                           | float | -20.00                                                  | Minium of y of the filter effective range     |
+| `intensity_threshold`             | float | 1.0                                                     | The threshold of average intensity for filter |
```

Comment:
Intensity will become uint8 for later topic interface, so we need to change this parameter or rename it or use 255*intensity_threshold to decide actual intensity threshold.